tests/unit/test_grid.py

```diff
@@ -139,6 +139,84 @@ def test_huge_cart_2d(self):
         self.assertTrue(np.allclose(g.face_centers, known_f_centr))
         self.assertTrue(np.allclose(g.cell_centers, known_c_centr))
 
+    def test_inconsistent_orientation_2d(self):
```

Comment:
Thanks for including tests!

That said, please include a line or two in each test to indicate what is being tested, and when meaningful how to interpret the failure of a test.

Reply:
We have included descriptive comments in the newly added tests